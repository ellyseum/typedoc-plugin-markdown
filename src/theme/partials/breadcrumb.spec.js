const context = {
  parent: {},
  name: 'name',
  url: 'url',
};

describe(`breadcrumb.hbs`, () => {
  test('should compile', () => {
    const result = global.compileTemplate('breadcrumb.hbs', context);
    expect(result).toMatchSnapshot();
  });
});
