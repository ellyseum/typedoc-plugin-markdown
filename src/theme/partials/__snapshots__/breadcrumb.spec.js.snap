// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breadcrumb.hbs should compile 1`] = `"[breadcrumb] > [name](../url)"`;
